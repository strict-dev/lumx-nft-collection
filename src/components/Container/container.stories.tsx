import { Meta, Story } from '@storybook/react'

import Container from '.'

export default {
  title: 'Container',
  component: Container
} as Meta

export const Default: Story = () => <Container>Hello</Container>
