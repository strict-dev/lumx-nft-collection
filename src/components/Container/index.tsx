type ContainerProps = {
  children: React.ReactNode
  className?: string
}

const Container: React.FC<ContainerProps> = ({ children, className }) => {
  return <div className={`mx-4 ${className ? className : ''}`}>{children}</div>
}

export default Container
