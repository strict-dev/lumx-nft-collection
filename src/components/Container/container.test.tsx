import { render } from '@testing-library/react'

import Container from '.'

describe('<Container />', () => {
  it('should render the heading', () => {
    const { container } = render(<Container>Hello</Container>)

    expect(container.firstChild).toMatchSnapshot()
  })
})
