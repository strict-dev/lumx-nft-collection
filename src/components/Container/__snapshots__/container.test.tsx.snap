// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render the heading 1`] = `
<div
  class="mx-4 "
>
  Hello
</div>
`;
